library/entity/multipart/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	name = "Quilt Multipart Entity API"
+	moduleName = "multipart"
+	id = "quilt_multipart"
+	description = "An API to support custom multipart entities."
+	library = "entity"
+	moduleDependencies {
+		core {
+			api("qsl_base")
+		}
+	}
+	injectedInterface("net/minecraft/class_1937") {
+		values = ["org/quiltmc/qsl/multipart/impl/EntityPartTracker"]
```

Comment:
A little less nitpicky nitpick: Those interfaces to be injected? Add an `@InjectedInterface` annotation to all of them